config/optional/block.block.localgov_alert_banner_base.yml

```diff
@@ -5,7 +5,7 @@ dependencies:
     - localgov_alert_banner
   theme:
     - localgov_base
-id: localgov_alert_banner
+id: localgov_alert_banner_base
 theme: localgov_base
 region: banner
 weight: 0
```

Comment:
Is renaming this file going to effect existing sites that have installed this module?
Need to test this with Config Distro to check.